controller_manager:
  ros__parameters:
    update_rate: 1000  # Hz

    use_sim_time: true

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
      use_sim_time: true

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController
      publish_rate: 1000  # Hz
      use_sim_time: true


joint_trajectory_controller:
  ros__parameters:
    use_sim_time: true

    joints:
      - j_c1_rf       #rf = leg 1
      - j_thigh_rf
      - j_tibia_rf
      - j_c1_rm       #rm = leg 2
      - j_thigh_rm
      - j_tibia_rm
      - j_c1_rr
      - j_thigh_rr    #rr = leg 3
      - j_tibia_rr
      - j_c1_lf
      - j_thigh_lf    #rr = leg 4
      - j_tibia_lf
      - j_c1_lm
      - j_thigh_lm    #rr = leg 5
      - j_tibia_lm
      - j_c1_lr
      - j_thigh_lr    #rr = leg 6
      - j_tibia_lr

    command_interfaces:
      - position

    state_interfaces:
      - position
      - velocity

